src/routes/phishing/preise/index.tsx

```diff
@@ -0,0 +1,241 @@
+        currency: 'EUR'
+      });
+      const silberPerUserPerMonth = bronzePerUserPerMonth * 1.3;
+      const silberPerMonth = silberPerUserPerMonth * userCount;
+      const silberPerYear = silberPerMonth * 12;
+
+      silverPerUserElement.innerText = silberPerUserPerMonth.toLocaleString(
+        'de-DE',
+        { style: 'currency', currency: 'EUR' }
+      );
+
+      silverPerMonthElement.innerText = silberPerMonth.toLocaleString('de-DE', {
+        style: 'currency',
+        currency: 'EUR'
+      });
+
+      silverPerYearElement.innerText = silberPerYear.toLocaleString('de-DE', {
+        style: 'currency',
+        currency: 'EUR'
+      });
+
+      const goldPerUserPerMonth = bronzePerUserPerMonth * 2;
+      const goldPerMonth = goldPerUserPerMonth * userCount;
+      const goldPerYear = goldPerMonth * 12;
+
+      goldPerUserElement.innerText = goldPerUserPerMonth.toLocaleString(
+        'de-DE',
+        {
+          style: 'currency',
+          currency: 'EUR'
+        }
+      );
+
+      goldPerMonthElement.innerText = goldPerMonth.toLocaleString('de-DE', {
+        style: 'currency',
+        currency: 'EUR'
+      });
+
+      goldPerYearElement.innerText = goldPerYear.toLocaleString('de-DE', {
+        style: 'currency',
+        currency: 'EUR'
+      });
+    }
+    function calculateBronzePricing(wantedUsers: number) {
+      wantedUsers = wantedUsers < 1 ? 1 : wantedUsers;
+      const pricingBronze = [
+        { user: 1, price: 20 },
+        { user: 5, price: 5 },
+        { user: 10, price: 3 },
+        { user: 25, price: 2.75 },
+        { user: 50, price: 2.5 },
+        { user: 75, price: 1.5 },
+        { user: 100, price: 1 },
+        { user: 200, price: 0.93 },
+        { user: 250, price: 0.84 },
+        { user: 500, price: 0.6 },
+        { user: 750, price: 0.5 },
+        { user: 1000, price: 0.4 },
+        { user: 2000, price: 0.38 },
+        { user: 3000, price: 0.36 },
+        { user: 4000, price: 0.34 },
+        { user: 5000, price: 0.32 },
+        { user: 6000, price: 0.3 }
+      ];
+
+      const lastElement = pricingBronze[pricingBronze.length - 1];
+
+      if (wantedUsers > lastElement.user) return lastElement.price;
+
+      let lastBigger = pricingBronze[0];
+      let lastSmaller = pricingBronze[0];
+
+      for (let i = 1; i < pricingBronze.length; i++) {
+        lastBigger = pricingBronze[i];
+        if (pricingBronze[i].user > wantedUsers) break;
+        lastSmaller = pricingBronze[i];
+      }
+
+      const preis = calculatePointOnLine(
+        lastSmaller.user,
+        lastSmaller.price,
+        lastBigger.user,
+        lastBigger.price,
+        wantedUsers
+      );
+      return preis;
+    }
+
+    function calculatePointOnLine(
+      x1: number,
+      y1: number,
+      x2: number,
+      y2: number,
+      x: number
+    ) {
+      const slope = (y2 - y1) / (x2 - x1);
+      const yIntercept = y1 - slope * x1;
+      return slope * x + yIntercept;
```

Comment:
Mir ist gerade noch ein Fehler aufgefallen, falls man bei genau 6000 Benutzern, also dem letzten festgelegten Preissegment ist.
Hier würde für die Steigung NaN herauskommen, da durch 0 geteilt wird.
Entweder man muss den NaN Fehlerfall behandeln oder vorher auf 0 prüfen.

```suggestion
      let slope = (y2 - y1) / (x2 - x1);
      if (isNaN(slope)) slope = 0;
      const yIntercept = y1 - slope * x1;
      return slope * x + yIntercept;
```